libs/blocks/table/table.css

```diff
@@ -456,6 +456,87 @@
   }
 }
 
+.table.has-addon .row-heading .col.col-heading {
+  display: flex;
```

Comment:
This seems to be the culprit of the issue from my previous screenshot, as it collides with `.table:not(.merch) .col-heading.col-1` which is hidden on mobile, some tweaks should be made to take this into account.

Reply:
okay I blocked styling on the first column in the header to avoid this collision. Let me know if that works.